### DISCUSSION TITLE
any suggestions on how to access my install after reCAPTCHA locked me out

### DISCUSSION THREAD

**Discussion by aloscher** [vanjarosoftware/Vanjaro.Platform#1647] [Q&A] [answered]
I added reCAPTCHA to my install and now I can no longer log in. I am thinking it is a firewall issue with my host so in the meantime, I would like to remove the reCAPTCHA from authenticating my login.

Any ideas on how to fix this through the DB?

Thanks in advance

**Comment by mksmanish79** [comment]
We are able to duplicate the issue on edge browser. Its working on chrome. Which browser you are using?

We are working on this issue and will be fixed soon.

"Any ideas on how to fix this through the DB?"
Below is the query to disable captcha from database.

update HostSettings set SettingValue='False' where SettingName='Vanjaro.Integration.Captcha.Enabled'
update PortalSettings set SettingValue='False' where SettingName='Vanjaro.Integration.Captcha.Enabled'

**Reply by aloscher** [reply]
> 
> 
> We are able to duplicate the issue on edge browser. Its working on chrome. Which browser you are using?
> 
> We are working on this issue and will be fixed soon.
> 
> "Any ideas on how to fix this through the DB?"
> Below is the query to disable captcha from database.
> 
> update HostSettings set SettingValue='False' where SettingName='Vanjaro.Integration.Captcha.Enabled'
> update PortalSettings set SettingValue='False' where SettingName='Vanjaro.Integration.Captcha.Enabled'

The fix did not work. Same message. I still see the recaptcha on the login screen

![image](https://user-images.githubusercontent.com/21082075/130339159-b7e9d36d-b685-453d-88fb-1fbbf167b58e.png)

**Reply by mksmanish79** [reply]
Its may be due to cached. Restart your application from iis and try again.

**Reply by aloscher** [reply]
> 
> 
> Its may be due to cached. Restart your application from iis and try again.

My hosting does not allow iis restart but I was able to log in after 24 hours or so. Thanks for your help

**Comment by scottw12** [comment]
I have had this problem on Chrome. However I upgraded an existing DNN site then added the Vanjaro_for_ DNN module.
Site works fine but when I attempt to login the sign-in button does nothing.


login:1 [DOM] Input elements should have autocomplete attributes (suggested: "current-password"): (More info: https://goo.gl/9p2vKq) 
input#Password
2
login:228 Uncaught ReferenceError: validateCaptcha is not defined
    at HTMLAnchorElement.onclick (login:228)
Navigated to https://dev.claytonagri-mktg.com/login
js:2 JQMIGRATE: Migrate is installed, version 3.2.0
200864640925053:4 [Violation] Avoid using document.write(). https://developers.google.com/web/updates/2016/08/removing-document-write
200864640925053:10 [Violation] Avoid using document.write(). https://developers.google.com/web/updates/2016/08/removing-document-write
login:1 [DOM] Input elements should have autocomplete attributes (suggested: "current-password"): (More info: https://goo.gl/9p2vKq) 
jotform.forms.js?3.3.24922:115 [Violation] Added synchronous DOM mutation listener to a 'DOMNodeInserted' event. Consider using MutationObserver to make the page more responsive.
element_main.js:446 [Violation] Avoid using document.write(). https://developers.google.com/web/updates/2016/08/removing-document-write
login:228 Uncaught ReferenceError: validateCaptcha is not defined
    at HTMLAnchorElemen

**Reply by scottw12** [reply]
The SettingName indicated in the instructions to update does not exist on my system in either table.

update HostSettings set SettingValue='False' where SettingName='Vanjaro.Integration.Captcha.Enabled'
update PortalSettings set SettingValue='False' where SettingName='Vanjaro.Integration.Captcha.Enabled'

**Comment by aloscher** [comment, marked as answer]
Ultimately, I believe my issue was firewall-related when I installed reCAPTCHA.
I had to disable the reCAPTCHA in the DB. The reCAPTCHA was still showing on my login page and because I could not log in,  I could not restart my Application to remove my Cache. I waited for 24 hours and my login started to work.
